src/vm/jitinterfacegen.cpp

```diff
@@ -246,9 +248,11 @@ void InitJITHelpers1()
 
                 ECall::DynamicallyAssignFCallImpl(GetEEFuncEntryPoint(AllocateStringFastMP), ECall::FastAllocateString);
             }
+#endif // FEATURE_IMPLICIT_TLS
         }
         else
         {
+#ifndef FEATURE_IMPLICIT_TLS
```

Comment:
Yea makes sense. Maybe I can also check for FEATURE_IMPLICIT_TLS for the ones that do access TLS and error in the remaining case.